### PR TITLE
[WIP] fixing path slash for windows + started RU translation

### PR DESCRIPTION
**Pull request by sn0wil**
#126
This mistake creates slug = undefiend and throw Error in application in Windows

### CODE REVIEW

**Comment by rigor789** [comment]
Looking good! Let me know if you want me to merge this, and then you can open a new pr for further changes, or I can wait and merge later.

**Comment by sn0wil** [comment]
@rigor789 I will inform you when `getting-started` will be done. After it I think you could make first merge :)

**Comment by sn0wil** [comment]
@rigor789 `getting-started` has been translated. Can you merge please?

File: content/index_ru.ejs [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+                <h1 class="text-5xl leading-tight text-white">Поистине нативные приложения с использованием Vue.js и NativeScript</h1>
+
+                <p class="w-4/5 leading-loose text-white mt-4">
+                    Хотите реализовать приложение, о котором мечтали, 
+                    без утомительного изучения новых языков?
+
+                    NativeScript-Vue - это лучший выбор для вас!
+                </p>
+
+                <a href="/en/docs/introduction"
```
Comment (rigor789) [inline content/index_ru.ejs]: change the url to `/ru/docs/introduction`

File: content/docs/ru/getting-started/4-templates.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+---
+title: Шаблоны
+contributors: [rigor789, tralves, jlooper, lewebsimple, sn0wil]
+outdated: false
+---
+
+Эта страница сожержит обзор поддерживаемых в настоящий момент шаблонов, которые вы можете использовать для начала работы с NativeScript-Vue. Это также подразумевает, что [подготовили вашу систему для разработки с NativeScript](/en/docs/getting-started/installation)
```
Comment (rigor789) [inline content/docs/ru/getting-started/4-templates.md]: Change the url to start with `/ru`

File: content/docs/ru/getting-started/5-nativescript-plugins.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+Использование UI плагинов очень похоже на использование NativeScript UI плагинов в Angular приложении. Рассмотрим пример использования [nativescript-gradient](https://github.com/EddyVerbruggen/nativescript-gradient), который используется в примере [listview sample](https://github.com/rigor789/nativescript-vue/tree/master/samples/app/app-with-list-view.js):
+ 
+Установите плагин, используя NativeScript CLI:
+
+```shell
+$ npm install --save nativescript-gradient
+```
+
+**Важно:** Если вы используете [vue-cli-template](/en/docs/getting-started/templates/#nativescript-vuevue-cli-template) вам может потребоваться запустить:
```
Comment (rigor789) [inline content/docs/ru/getting-started/5-nativescript-plugins.md]: Change to `/ru`

**Comment by rigor789** [comment]
Another thing, change the `contributors` for the files you've fully translated to only include your GitHub username

**Comment by sn0wil** [comment]
@rigor789 , links have been fixed. I am in contrubutors list in these files

**Comment by rigor789** [comment]
@sn0wil Alright, but what I'm saying is that you can remove everyone else from `contributors` if you translated the page completely.

**Comment by sn0wil** [comment]
@rigor789 Oh, okay. Working on it. I thought that I must save other contributors :)

**Comment by sn0wil** [comment]
I will make this in another PR

**Pull request merged**